django/views/templates/default_urlconf.html

```diff
@@ -65,213 +50,113 @@
-            animation: burnInner2 .1s infinite ease-in-out alternate;
-          }
-          @-webkit-keyframes burnInner2 {
-            0% {
-              -webkit-transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
-              transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
-            }
-            100% {
-              -webkit-transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
-              transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
-            }
+            animation: burnInner2 .1s 70 ease-in-out alternate;
           }
           @keyframes burnInner2 {
             0% {
-              -webkit-transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
-              transform: translate(265px, 249px) scale(1.9) rotate(180deg) skew(5deg);
+              transform: translate3d(0, 0, 0);
             }
             100% {
-              -webkit-transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
-              transform: translate(265px, 253px) scale(2.2) rotate(180deg) skew(-5deg);
+              transform: translate3d(0, 3px, 0);
             }
           }
-          @-webkit-keyframes thurst {
+          @keyframes thurst {
             0% {
               opacity: 1;
             }
             100% {
               opacity: .5;
             }
           }
-          @keyframes thurst {
-            0% {
-              opacity: 1;
-            }
-            100% {
-              opacity: .5;
+          @media (prefers-reduced-motion: reduce) {
+            .exhaust__line,
+            .smoke,
+            .flame {
+              animation: none;
             }
           }
-          h2 {
-            font-size: 22px;
-            max-width: 500px;
+          h1 {
+            font-size: 1.375rem;
+            max-width: 32rem;
             margin: 5px auto 0;
           }
           main p {
-            font-size: 16px;
-            line-height: 20px;
-            max-width: 390px;
+            line-height: 1.25;
+            max-width: 25rem;
             margin: 15px auto 0;
-            color: #888888;
           }
           footer {
             padding: 25px 0;
             position: fixed;
             left: 50%;
             bottom: 0;
             width: 960px;
-            -webkit-transform: translateX(-50%);
             transform: translateX(-50%);
-            -webkit-transform-style: preserve-3d;
             transform-style: preserve-3d;
             border-top: 1px solid #efefef;
           }
           .option {
+            display: block;
             float: left;
             width: 33.33%;
             box-sizing: border-box;
             padding-right: 5px;
+            text-decoration: none;
           }
           .option svg {
-            width: 25px;
-            height: 25px;
+            width: 1.5rem;
+            height: 1.5rem;
```

Comment:
This is totally optional, thought it would look nicer for the icons to look bigger when text is larger.